plugins/hypervisors/kvm/src/com/cloud/hypervisor/kvm/resource/wrapper/LibvirtModifySshKeysCommandWrapper.java

```diff
@@ -52,7 +52,7 @@ public Answer execute(final ModifySshKeysCommand command, final LibvirtComputing
         String result = null;
         if (!sshKeysDir.exists()) {
             // Change permissions for the 700
-            final Script script = new Script("mkdir", libvirtComputingResource.getTimeout(), s_logger);
+            final Script script = new Script("mkdir", libvirtComputingResource.getTimeout().getMillis(), s_logger);
```

Comment:
Please consider adding an overridden version of the `Script(String, int, Logger)` constructor that accepts `Duration` to encapsulate this type conversion.